changelog.d/8169.feature

```diff
@@ -0,0 +1 @@
+Use Type=notify in debian systemd service.
```

Comment:
changes to the debian packaging go in the `debian/changelog` file, not here: please could you move this?

(there are some notes in CONTRIBUTING.rst)